WordPressKit/ReaderSiteSearchServiceRemote.swift

```diff
@@ -0,0 +1,84 @@
+import Foundation
+import WordPressShared
+
+public class ReaderSiteSearchServiceRemote: ServiceRemoteWordPressComREST {
+
+    public enum ResponseError: Error {
+        case decodingFailure
+    }
+
+    /// Searches Reader for sites matching the specified query.
+    ///
+    /// - Parameters:
+    ///     - query: A search string to match
+    ///     - offset: The first N results to skip when returning results.
+    ///     - count: Number of objects to retrieve.
+    ///     - success: Closure to be executed on success. Is passed an array of
+    ///                ReaderFeeds, a boolean indicating if there's more results
+    ///                to fetch, and a total feed count.
+    ///     - failure: Closure to be executed on error.
+    ///
+    public func performSearch(_ query: String,
+                                   offset: Int = 0,
+                                   count: Int,
+                                   success: @escaping (_ results: [ReaderFeed], _ hasMore: Bool, _ feedCount: Int) -> Void,
+                                   failure: @escaping (Error) -> Void) {
```

Comment:
OKAY this is SO NIPTICKY it's actually shameful to jot it down.

Feel absolutely free to skip: the inconsistent left spacing with the first parameter ... err... bugs me? (SORRY)

Reply:
FIXED!